ibm/mas_devops/roles/cp4d_db2wh_backup/tasks/main.yml

```diff
@@ -0,0 +1,126 @@
+---
+# 1. Provide debug information to the user
+# -----------------------------------------------------------------------------
+- name: "DB2 Backup - Debug information"
+  debug:
+    msg:
+      - "CPD Namespace .......................... {{ cpd_meta_namespace_source }}"
+      - "DB2 Instance ID ........................ {{ db2wh_instance_id_source }}"
+      - "DB2 Backup Files Directory ............. {{ db2wh_backup_folder }}"
+
+# 2. Fail if no required keys are provided
+# -----------------------------------------------------------------------------
+- name: "Fail if cpd_meta_namespace_source has not been provided"
```

Comment:
Could you use the new/better convention using assert for these ... see issue https://github.com/ibm-mas/ansible-devops/issues/124

Example of them in use: https://github.com/ibm-mas/ansible-devops/blob/upgrade87/ibm/mas_devops/roles/suite_upgrade/tasks/main.yml#L5

Reply:
Fixed.